runtime/proto/service/log_replication.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package org.corfudb.runtime;
+option java_package = "org.corfudb.runtime";
+
+import "rpc_common.proto";
+
+message LogReplicationEntryMetadataMsg {
+  int64 siteConfigID = 1;
```

Comment:
here, can we rename this to topologyConfigId?

Reply:
Done.